src/components/Tooltip/Tooltip.tsx

```diff
@@ -20,7 +20,7 @@ export const Tooltip: React.FC<TooltipProps> = ({ content, children, disabled, .
         isDisabled = true;
     }
     return (
-        <Tippy content={content} disabled={isDisabled} {...rest}>
+        <Tippy content={content} disabled={isDisabled} maxWidth={320} arrow={false} {...rest}>
```

Comment:
I have difficulty checking in the dev-tools, but it seems to me that the font-size is still large (16px) not medium (14px) as required in the ticket. 

I think you need to make a scss file, and make styles there then apply here. We have a (new) pattern to apply typography. You should use `compose: OneUI-label-text from global`. Let me know if this is not clear and we can look at it together.

Reply:
![image](https://github.com/user-attachments/assets/80287104-6512-4021-af65-54b9da355808)
I did check the font size and it seems to be the way we wanted (14px)

Reply:
nice! thanks